api/src/main/java/com/velocitypowered/api/event/player/PlayerClientBrandEvent.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2018 Velocity Contributors
+ *
+ * The Velocity API is licensed under the terms of the MIT License. For more details,
+ * reference the LICENSE file in the api top-level directory.
+ */
+
+package com.velocitypowered.api.event.player;
+
+import com.velocitypowered.api.proxy.Player;
+
+/**
+ * Fired when a {@link Player} sends the <code>minecraft:brand</code> plugin message.
+ */
+public class PlayerClientBrandEvent {
+  private final Player player;
+  private final String brand;
+
+  /**
+   * Creates a new instance.
+   *
+   * @param player the {@link Player} of the sent client brand
+   * @param brand the sent client brand
+   */
+  public PlayerClientBrandEvent(Player player, String brand) {
+    this.player = player;
+    this.brand = brand;
```

Comment:
I believe there should be preconditions here, i.e. Preconditions.checkNotNull as the other events in the API use.